### PR TITLE
Support of the X509 header for any public key

### PR DESCRIPTION
**Pull request by DevNathan**
### **This PR fixes the issue #81.**

It enables to prepend a **X509 header** to a headerless public key. 
The new method does the exact opposite of the native method `stripKeyHeader()` 

If the given key isn't headerless, it is returned as is. 

A part of the new code come from [Heimdall](https://github.com/henrinormak/Heimdall )'s project released under *MIT Licence*. 

Header key can be verified on [Lapo's online decoder](https://lapo.it) by anyone.

With the new method, all public keys are compatible with the most of javascript, java, python (etc...) rsa algorithms. 
It could fix some of the gihtub SwiftyRSA's issues

### CODE REVIEW

**Comment by DevNathan** [comment]
@starback According to the ReadMe you're the only mantainer of this project. Could you, please, consider reviewing that PR ?
I also saw that @ldiqual was interested in a such feature (#81) and was able to review it. Are-you still able to do so ?

**Comment by DevNathan** [comment]
@pmacro You seem to be the last active contributor. Could you, please, consider reviewing that PR, in order push these features in the official package ?

**Review by nickm01 — COMMENTED** [review]
@DevNathan will review shortly thanks for your contribution!

**Comment by DevNathan** [comment]
Any updates ?

**Comment by DevNathan** [comment]
@nickm01 If you had enough time to review these changes, could  you consider  merging it ?

**Comment by nickm01** [comment]
> @nickm01 If you had enough time to review these changes, could you consider merging it ?

Look

File: Source/X509Certificate.swift [hunk 1]
```diff
@@ -0,0 +1,98 @@
+//
+//  X509Certificate.swift
```
Comment (nickm01) [inline Source/X509Certificate.swift:2]: @DevNathan is it possible to add a test for this?

**Comment by nickm01** [comment]
@DevNathan Also - could you update the README?  Thank you!

**Comment by DevNathan** [comment]
With pleasure ! I will work on these two points this week-end.

**Comment by DevNathan** [comment]
@nickm01 Is it good for you ?

**Review by nickm01 — APPROVED** [review]

**Pull request merged**